templates/reverse.j2

```diff
@@ -8,24 +8,24 @@ $TTL 1W
 	IN	NS	ns1.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 ;
 ; syntax is "last octet" and the host must have fqdn with trailing dot
-1       IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain }}.
+{{ helper.ipaddr.split('.')[3] }}	IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 
 {% for m in masters %}
-{{ m.ipaddr.split('.')[3] }}	IN	PTR	{{ m.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.
+{{ m.ipaddr.split('.')[3] }}	IN	PTR	{{ m.name | lower }}.
 {% endfor %}
 ;
-{{ bootstrap.ipaddr.split('.')[3] }}	IN	PTR	{{ bootstrap.name | lower  }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.
+{{ bootstrap.ipaddr.split('.')[3] }}	IN	PTR	{{ bootstrap.name | lower  }}.
```

Comment:
This won't work. it needs the fqdn if using the `.` delimiter.

Reply:
Discovered in my install that bootstrap.name is a FQDN.